include/kyosu/functions/erf.hpp

```diff
@@ -57,8 +57,7 @@ namespace kyosu
 //!   @code
 //!   namespace kyosu
 //!   {
-//!      template<eve::ordered_value T>       constexpr auto erf(T z) noexcept;  //1
-//!      template<kyosu::concepts::complex T> constexpr auto erf(T z) noexcept;  //2
+//!      constexpr auto erf(auto z) noexcept;
```

Comment:
Looks liek you mangled a bunch of those definitions, can you recheck the doc texts ?

Reply:
ok